changelog/next/features/4918--platform-tls-options.md

```diff
@@ -0,0 +1,3 @@
+The `platform` plugin now understands the `skip-peer-verification`
+and `cacert` options in order to enable connections to self-hosted
+platform instances with self-signed TLS certificates.
```

Comment:
These should be documented for the Sovereign Edition, probably in some place where the platform setup is described.

Should we support `tenzir.platform-foo` in addition to `plugin.platform.foo`, so that users can set `TENZIR_PLATFORM_SKIP_PEER_VERIFICATION` and `TENZiR_PLATFORM_CACERT`?